### PR TITLE
fix-experiment-control-nits

### PR DESCRIPTION
**Pull request by johngruen**
This hides the "send link to device" button for mobile clients and generally tidies up the experiment control UI

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3641/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/7af0ef600d53ae0b702c9d2e59fe58a1a2e48233/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3641/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/cb7ad2a986d9beedb4441703728f867f1a902a19/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3641/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/0e500a384a4dde456af844ea637268278a2b451f/

File: frontend/src/app/containers/ExperimentPage/ExperimentControls.js [hunk 1]
```diff
@@ -128,14 +128,19 @@ export default function ExperimentControls({
...
-      <Localized id="mobileDialogLaunchButton">
-        <a
-          className="button default"
-          onClick={doShowMobileAppDialog}>Send App Link to Device</a>
-      </Localized>
-    );
+  if (!isMobile(userAgent)) {
+    if (platforms.includes("ios") || platforms.includes("android")) {
+      buttons.unshift(
+        <Localized id="">
```
Comment (pdehaan) [inline frontend/src/app/containers/ExperimentPage/ExperimentControls.js]: Does this need an `id`?

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3641/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/64678ed6ab4e29919de7ea0935cb07d44507660d/

**Review by meandavejustice — APPROVED** [review]

**Pull request merged**